README.md

```diff
@@ -118,6 +118,7 @@ For optional settings check the [docs](https://hexdocs.pm/sentry/readme.html).
 | `source_code_path_pattern` | False  | `"**/*.ex"` | |
 | `filter` | False | | Module where the filter rules are defined (see [Filtering Exceptions](https://hexdocs.pm/sentry/Sentry.html#module-filtering-exceptions)) |
 | `json_library` | False | `Jason` | |
+| `log_level` | False | `:warn` | |
```

Comment:
If you could add a short note about what the config is used for, that would be super.  Something along the lines of `This sets what level Sentry will log at when it fails to send due to an invalid event or API error`

Reply:
Done.